src/app/features/test-records/amend/amend-test-records-routing.module.ts

```diff
@@ -48,18 +50,56 @@ const routes: Routes = [
           },
           {
             path: 'amend-test-details',
-            component: TestRecordComponent,
+            component: TestRouterOutletComponent,
             data: { title: 'Test details', roles: Roles.TestResultAmend },
             resolve: { load: TestResultResolver, testTypeTaxonomy: TestTypeTaxonomyResolver, defectTaxonomy: DefectsTaxonomyResolver },
-            canActivate: [RoleGuard]
+            canActivate: [RoleGuard],
```

Comment:
Because of the change to the routing module, the `TestRecordComponent` is no longer a child of the `TestResultsSummaryComponent`, but a granchild. This means that when we submit an amendment, the `backToTestRecord` method in the `TestRecordComponent` should navigate to `['../..']` rather than `['..']` as it is right now